packages/devtools_app/test/cpu_profiler/cpu_profiler_test.dart

```diff
@@ -2,24 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
Was this import intentional?

Reply:
yeah, devtools_app.dart import is okay for tests, just not in src/